src/test/kotlin/no/skatteetaten/aurora/boober/unit/AuroraDeploymentSpecValidatorTest.kt

```diff
@@ -390,20 +390,38 @@ class AuroraDeploymentSpecValidatorTest : AbstractAuroraConfigTest() {
         }
     }
 
+    @Test
+    fun `Fails when the name of the the secretVault is to long`() {
+
+        auroraConfigJson["utv/aos-simple.json"] = """{
+            |"secretVaults": {
+            |  "this-is-way-more-then-63-characters-long-secretVault-name" : { }
+            |  }
+            |}""".trimMargin()
+        val deploymentSpec = createDeploymentSpec(auroraConfigJson, DEFAULT_AID)
+
+        assertThat {
+            specValidator.validateSecretNames(deploymentSpec)
+        }.thrownError {
+            isInstanceOf(AuroraDeploymentSpecValidationException::class)
+            hasMessage("The name of the secretVault=aos-simple-this-is-way-more-then-63-characters-long-secretvault-name is too long. Max 63 characters. Note that we prefix the vaultName with @name@- it does not already")
```

Comment:
Tror jeg ville gjort denne asserten litt mer relaxed, kanskje sjekke at navnet er med i stedet for hele meldingen?